src/app/app.module.ts

```diff
@@ -5,20 +5,22 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { SharedModule } from './shared/shared.module';
-import { HeaderComponent } from './header/header.component';
+import { MainInformationModule } from './section-main/main-information/main-information.module';
+import { SectionMainModule } from './section-main/section-main.module';
+import { HeaderModule } from './shared/header/header.module';
 
 @NgModule({
-  declarations: [
-    AppComponent,
-    HeaderComponent,
-  ],
+  declarations: [AppComponent],
   imports: [
     BrowserModule,
     AppRoutingModule,
     BrowserAnimationsModule,
-    SharedModule
+    SharedModule,
+    MainInformationModule,
+    SectionMainModule,
```

Comment:
Remove modules from main module. They should be lazy loaded.

Reply:
fix